qadence/types.py

```diff
@@ -470,16 +470,25 @@ class AnalogNoise(StrEnum):
     DEPHASING = "Dephasing"
 
 
+class ReadoutNoise(StrEnum):
+    """Type of readout protocol."""
+
+    INDEPENDENTREADOUT = "IndependentReadout"
```

Comment:
It becomes a little verbose to specify READOUT.INDEPENDENTREADOUT and READOUT.CORRELATEDREADOUT. 

Maybe remove the readout at the end, so that they can be specified like READOUT.INDEPENDENT and READOUT.CORRELATED.